libr/util/print_code.c

```diff
@@ -6,13 +6,13 @@
 static const char* bits_to_c_code_fmtstr(int bits) {
```

Comment:
Isnt this firmat string used against int bits? Why does it needs to be 64? Maybe the fix is in the caller

Reply:
https://github.com/radareorg/radare2/blob/56cdc7c112e5a51ce71715dcf84252d016155076/libr/util/print_code.c#L26-L46
https://github.com/radareorg/radare2/blob/56cdc7c112e5a51ce71715dcf84252d016155076/libr/util/print_code.c#L72-L83
These are the only 2 places that call `bits_to_c_code_fmtstr`. `fmtstr` is used to format the return value of r_read_ble, which is ut64. Either modify like this PR, or do `(int)r_read_ble` are OK, up to you.

Reply:
as long as the argument is "int" i would expect the function to return a format string for `int`.  so casting is probably better and more implicit. can you update this. also its a waste of bits to represent the number 64 in 64bits :D

Reply:
Oh, I was wrong. Line 83 we cannot know the value of `bits`, so we cannot do the cast. Have to waste bits here.

> as long as the argument is "int" i would expect the function to return a format string for `int`. so casting is probably better and more implicit. can you update this. also its a waste of bits to represent the number 64 in 64bits :D

Reply:
a value bigger than 64 shouldnt be valid, to specify the bits. maybe at some point we may have 128 or 256. but none of this value need 64bit variables to hold this. I mean i understand the fix, but i dont want to waste that much bits and use bigger format strings for that case. You can use & UT32_MAX to wrap the 64bit value and just get the lowerbits to get the format string if that's what worrying you.